app/models/solidus_subscriptions/subscription.rb

```diff
@@ -434,11 +434,11 @@ def emit_events_for_update
     end
 
     def self.ransackable_attributes(_auth_object = nil)
-      %w[actionable_date billing_address_id created_at currency end_date guest_token id id_value interval_length interval_units paused payment_method_id payment_source_id payment_source_type shipping_address_id skip_count state store_id successive_skip_count updated_at user_id]
+      %w[actionable_date created_at end_date state  updated_at user_id]
     end
 
     def self.ransackable_associations(_auth_object = nil)
-      %w[billing_address events installment_details installments line_items orders payment_method payment_source shipping_address store user]
```

Comment:
Can you explain why you removed lots of those associations? I do not know much about this extension and it's relations, but it seems beneficial to me to be able to search a `bill_address`, `ship_address` or `line_items`.

Reply:
Yeah sure. Currently, you can search by date range (created_at/updated_at), actionable date, state, and email (via user) in the subscriptions list page. Similar adjustments in installments and events pages.

Refer this view file for Subscriptions list page `app/views/spree/admin/subscriptions/index.html.erb`